core/src/test/java/com/linecorp/armeria/server/HttpServerPathTest.java

```diff
@@ -121,7 +121,11 @@ private static void urlPathAssertion(HttpStatus expected, String path) throws Ex
         try (Socket s = new Socket(NetUtil.LOCALHOST, server.httpPort())) {
             s.setSoTimeout(10000);
             s.getOutputStream().write(requestString.getBytes(StandardCharsets.US_ASCII));
-            assertThat(new String(ByteStreams.toByteArray(s.getInputStream()), StandardCharsets.US_ASCII))
+            final BufferedReader in = new BufferedReader(
```

Comment:
Due to the following implementation, the `ByteStreams.toByteArray(s.getInputStream())` call returns the value when the inputStream is closed when the connection is closed from the server.
https://github.com/google/guava/blob/master/guava/src/com/google/common/io/ByteStreams.java#L195~L198

With this PR, I changed the server to wait for the client to close first instead of closing the connection immediately. However in this test, the client receives the `Connection: Close` header from the server but ignores it. So the test waits for the server to close the connction, causing the test timeout.
I tried to adjust the `defaultHttp1ConnectionCloseDelayMillis` using `@SetSystemProperty` or `System.setProperty()` to reduce the test time, but it didn't work well in shadedTest 😭 
Since only the first line is needed for this test validation, I changed it to read only the first line and check the result to make the test take less time.
If it is a problem, I would consider reducing the `defaultHttp1ConnectionCloseDelayMillis`.